CHANGELOG.md

```diff
@@ -5,12 +5,15 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Added
 - Github workflow for changelog verification ([#239](https://github.com/opensearch-project/opensearch-java/pull/239))
 - Github workflow for dependabot PRs ([#247](https://github.com/opensearch-project/opensearch-java/pull/247))
+- Github workflow for dependabot PRs ([#234](https://github.com/opensearch-project/opensearch-java/pull/234))
```

Comment:
This one slipped in, bad copy-paste, remove.

Reply:
Sorry, I just thought it was needed there. Fixed it.